File: R/tests/testthat/test_pstRT_rlddm1.R
context("Test pstRT_rlddm1")
library(hBayesDM)

test_that("Test pstRT_rlddm1", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(pstRT_rlddm1(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_pstRT_rlddm1.py
import pytest

from hbayesdm.models import pstRT_rlddm1


def test_pstRT_rlddm1():
    _ = pstRT_rlddm1(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
